webpack.config.js

```diff
@@ -21,7 +21,7 @@ module.exports = {
     filename: 'bundle.js'
   },
   resolve: {
-    modules: ['node_modules', 'src']
+    modules: ['node_modules', 'src', 'lib']
```

Comment:
Added `lib`, so we could add standalone libs (there is no npm package for `session.js` (https://github.com/codejoust/session.js/)